package.json

```diff
@@ -30,6 +30,7 @@
 		"lint": "eslint 'src' '.storybook' --ext .ts,.tsx,.js,.jsx,.json,.d.ts",
 		"lint:fix": "eslint 'src' '.storybook' --ext .ts,.tsx,.js,.jsx,.json,.d.ts --fix",
 		"build:tokens": "(git -C my-axiory-tokens pull || git clone https://github.com/purple-technology/my-axiory-tokens.git) && node style-dictionary-build && eslint 'src/tokens/' --ext .json --fix",
+		"tokens-temp": "node style-dictionary-build && eslint 'src/tokens/' --ext .json --fix",
```

Comment:
proc se to prosim jmenuje temp? :)

Reply:
It's been there by accident. I noticed it, was about to remove it but then something interrupted me and I forgot to commit and push it 😊